src/tablestore.rs

```diff
@@ -388,21 +413,17 @@ impl TableStore {
                         blocks_read.push_back(cached_block);
                     }
                     None => {
-                        // If a block is not in cache, mark the start of an uncached range
```

Comment:
Can you leave these comments? This code is somewhat complex, and I prefer more verbose comments for it.